language/polkavm-move-native/src/host.rs

```diff
@@ -43,37 +42,49 @@ impl MemAllocator {
     }
 
     /// Allocate guest memory in the auxiliary data region.
-    pub fn alloc(&mut self, size: usize, _align: usize) -> Result<u32, MemoryAccessError> {
-        debug!(
-            "Allocating {} bytes in guest memory at offset {}",
-            size, self.offset
-        );
-        if self.offset as usize + size > self.size as usize {
+    pub fn alloc(&mut self, size: usize, align: usize) -> Result<u32, MemoryAccessError> {
+        let align = align.max(1);
```

Comment:
Isn't it safer to align with 4 (32-bit) or 8 (64-bit) instead of 1.

Reply:
I don't think alignment matters much in the PVM. If alignment is passed by the global allocator, it will use that, this just ensures that it's not 0.